gprofiler/main.py

```diff
@@ -557,6 +490,28 @@ def parse_cmd_args():
     return args
 
 
+def _add_profilers_arguments(parser):
+    profilers_registry = get_profilers_registry()
+    for profiler_name, profiler_config in profilers_registry.items():
+        profiler_argument_group = parser.add_argument_group(profiler_name)
+        profiler_argument_group.add_argument(
+            f"--{profiler_name.lower()}-mode",
+            dest=f"{profiler_name.lower()}_mode",
+            default=profiler_config.default_mode,
+            help=profiler_config.profiler_mode_help,
+            choices=profiler_config.possible_modes,
+        )
+        profiler_argument_group.add_argument(
+            f"--no-{profiler_name.lower()}",
+            dest=profiler_name.lower(),
+            action="store_false",
+            default=True,
+            help=f"Deprecated. Use '--{profiler_name.lower()}-mode none' instead",
```

Comment:
You're actually adding new options by this (we didn't have `--no-perf`, for example). Can just remove the "deprecated" note if you are maintaining it here